src/workerd/io/compatibility-date.capnp

```diff
@@ -975,4 +975,11 @@ struct CompatibilityFlags @0x8f8c1b68151b6cef {
     $compatDisableFlag("disable_nodejs_http2_module")
     $impliedByAfterDate(name = "nodeJsCompat", date = "2025-09-01");
   # Enables the Node.js http2 module stubs.
+
+  experimentalAllowEvalAlways @113 :Bool
```

Comment:
We could just give it a good name and reuse this same flag in the future when we are actually ready to allow eval.
I'd rather not have two flags for the same thing